parent_child.js

```diff
@@ -4,15 +4,14 @@ var Checkbox = function(){
 Checkbox.prototype.bindEvents = function(){
  var Outerscope = this;
  $('.parentInputClass').click(function(){
- 	Outerscope.selected($(this));
+ 	Outerscope.togglecheckboxes($(this));
```

Comment:
the name of the method is not suitable, it does more than just toggling. Pls update. Also pls ensure that the indentation of your code is correct.